speech/cloud-client/src/main/java/com/example/speech/InfiniteStreamRecognize.java

```diff
@@ -66,15 +69,31 @@
   private static ByteString tempByteString;
 
   public static void main(String... args) {
+    InfiniteStreamRecognizeOptions options = InfiniteStreamRecognizeOptions.fromFlags(args);
+    if (options == null) {
+      // Could not parse.
```

Comment:
How about printing something instead of the comment?

Print error "Failed to parse options"

Otherwise someone might run the sample + it will exit without a message + they won't know what's going on